src/engine/SCons/Tool/msvc.py

```diff
@@ -271,6 +271,10 @@ def generate(env):
     env['SHOBJPREFIX']    = '$OBJPREFIX'
     env['SHOBJSUFFIX']    = '$OBJSUFFIX'
 
+    # MSVC probably wont support unistd.h so default
+    # without it for lex generation
+    env["LEXUNISTD"] = SCons.Util.CLVar("--nounistd")
```

Comment:
Looks like there's a flag "--wincompat" which should be used instead?

Reply:
--wincompat is only available in win_flex I believe, while --nounistd is in both. 

--wincompat is a bit better for MSVC, as it will replace some of things unistd.h brings in.

From win_flex [Changelog](https://github.com/lexxmark/winflexbison/blob/master/changelog.md):
```
* added --wincompat option to win_flex (this option changes `<unistd.h>` unix include with `<io.h>` windows analog
  also `isatty/fileno` functions changed to `_isatty/_fileno`)
```

Any ideas how to detect if win_flex will be used from msvc.py? I assume the user would need to set this if the generated source needs something from the <io.h> include.